packages/d3fc-webgl/test/shaders/line/shaderSpec.js

```diff
@@ -0,0 +1,39 @@
+import * as fc from '../../../index';
+import { expectVertexShader, getShaders } from '../../util/vertextShaderUtil';
+
+it('test', () => {
```

Comment:
Could you rename this to describe the test?

Reply:
Yeah, missed that 👍